apps/site/src/app/apply/sections/Title/Title.tsx

```diff
@@ -0,0 +1,10 @@
+import styles from "./Title.module.scss";
+
+export default function Title() {
+    return(
+        <>
+            <h1 className={`${styles.title} font-display text-[#fffce2] text-8xl text-center max-[500px]:text-7xl`}>Apply</h1>
```

Comment:
I feel "Apply" as a title of the page looks off. Maybe something like "IrvineHacks Application Form"?

Reply:
the "Apply" was on the figma page, and "IrvineHacks Application Form" is a tiny bit long in my opinion but I don't really know; I can change if it you think it would look best

Reply:
cc: @samderanova

Reply:
We should stick with Apply. I personally don't see an issue with it, especially since we used it last year.